packages/eslint-config-mavenlint/package.json

```diff
@@ -6,11 +6,10 @@
   "repository": "https://github.com/mavenlink/mavenlint.git",
   "license": "MIT",
   "peerDependencies": {
-    "eslint": ">=4.14.0",
-    "eslint-plugin-import": ">=2.8.0"
+    "eslint": ">=4.14.0"
   },
   "dependencies": {
     "eslint-config-airbnb": "15.1.0",
-    "eslint-plugin-jasmine": "^2.9.3"
+    "eslint-plugin-import": "^2.14.0"
```

Comment:
It looks like the version of `eslint-plugin-import` went down. Does that matter?

Reply:
On second look, I think it went up?